WooCommerce/Classes/Authentication/Jetpack Setup/Native Jetpack Setup/JetpackSetupViewModel.swift

```diff
@@ -168,9 +172,14 @@ private extension JetpackSetupViewModel {
             case .failure(let error):
                 DDLogError("⛔️ Error retrieving Jetpack: \(error)")
                 self.setupError = error
-                if self.hasEncounteredPermissionError == false,
-                    self.setupSteps.contains(.installation) {
-                    // plugin is likely to not have been installed, so proceed to install it.
+                if case .responseValidationFailed(reason: .unacceptableStatusCode(code: 404)) = error as? AFError {
```

Comment:
Nit: Kindly consider adding comments explaining what this error code means in this context. 

I like what you have [here](https://github.com/woocommerce/woocommerce-ios/blob/dfdd2255cbc7c3e79ff3f187936dc90420829a2d/WooCommerce/Classes/Authentication/Jetpack%20Setup/Native%20Jetpack%20Setup/JetpackSetupViewModel.swift#L65-L70) which explains the error well. 💯 Maybe we can add something like that as a private var. 😄

Reply:
I left some comments in the content of this check - I think that should be sufficient.